src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplSearchSteps.java

```diff
@@ -1,52 +1,54 @@
 package gov.healthit.chpl.aqa.stepDefinitions;
-
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
+import static org.testng.Assert.assertTrue;
 import gov.healthit.chpl.aqa.pageObjects.ListingDetailsPage;
 import gov.healthit.chpl.aqa.pageObjects.SearchPage;
-
 /**
- * Class DisclosureUrlSteps definition.
+ * Class DeleteDupListingsSteps definition.
  */
-public class DisclosureUrlSteps {
 
+public class ChplSearchSteps {
     private WebDriver driver;
     private static final int TIMEOUT = 30;
     private String url = System.getProperty("url");
 
     /**
      * Constructor creates new driver.
      */
-    public DisclosureUrlSteps() {
+    public ChplSearchSteps() {
         driver = Hooks.getDriver();
         if (StringUtils.isEmpty(url)) {
             url = "http://localhost:3000/";
-        }
-   }
+           }
+    }
+    /**
+     * Get user to CHPL search page.
+     */
+    @Given("^I am on CHPL search page$")
+    public void iAamOnCHPLSearchPage() {
+        driver.get(url);
+    }
     /**
-     * Loads a listing for given Database ID.
-     * @param dbId the Database Id of a listing to load
+     * @param chplId Search for a listing with given chplId.
      */
-    @Given("^I am on listing details page with database ID \"([^\"]*)\"$")
-    public void loadListingDetails(final String dbId) {
-        driver.get(url + "#/product/" + dbId);
-   }
+    @When("^I search for a listing with CHPL ID \"(.*)\"$")
+    public void searchForCHPLID(final String chplId) {
+        SearchPage.searchField(driver).sendKeys(chplId);
+    }
     /**
-     * Assert that Mandatory Disclosures URL exists and is correct.
-     * @param urlValue the URL value to assert
+     * Assert message when no results found.
      */
-    @Then("^the Mandatory Disclosures URL field should show Disclosures URL \"(.*)\"$")
-    public void verifyMandatoryDisclosureurlValue(final String urlValue) {
-        String actualText = ListingDetailsPage.disclosureUrl(driver).getText();
-        assertTrue(actualText.contains(urlValue), "Expect " + urlValue + " to be found in " + actualText);
-   }
+    @Then("^the search page shows 'No results found' message$")
+    public void verifyMessage() {
+        assertTrue(SearchPage.noResultsFound(driver).getText().contains("No results found"));
```

Comment:
With the "assertTrue" it's really helpful to have an error string that prints out, as otherwise all the report says was "expect false to be true", and that's not useful